src/common/packager.ts

```diff
@@ -43,7 +45,7 @@ export class Packager {
                             let args = ["--port", Packager.PORT];
                             let childEnvForDebugging = Object.assign({}, process.env, { REACT_DEBUGGER: "echo A debugger is not needed: " });
 
-                            Log.logMessage("Starting Packager", outputChannel);
+                            Log.logMessage("Starting Packager");
```

Comment:
So what is the default global output location for inside the extension process? Are we intentionally coalescing output from the packager and all other sources together now?